src/object.c

```diff
@@ -1184,9 +1184,9 @@ struct redisMemOverhead *getMemoryOverheadData(void) {
     mh->total_frag_bytes =
         server.cron_malloc_stats.process_rss - server.cron_malloc_stats.zmalloc_used;
     mh->allocator_frag =
-        (float)server.cron_malloc_stats.allocator_active / server.cron_malloc_stats.allocator_allocated;
+        (float)server.cron_malloc_stats.allocator_frag_smallbins_bytes / server.cron_malloc_stats.allocator_allocated + 1;
     mh->allocator_frag_bytes =
-        server.cron_malloc_stats.allocator_active - server.cron_malloc_stats.allocator_allocated;
+        server.cron_malloc_stats.allocator_frag_smallbins_bytes;
```

Comment:
so now both allocator_frag and allocator_frag_bytes are based on small-bins only, so why do we need the new allocator_frag_smallbins_bytes field?

Reply:
`allocator_frag_smallbins_bytes` is fetched once per second, but `allocator_frag` and `allocator_frag_bytes` are temporarily populated by `getMemoryOverheadData()`, they're not the same.
am i missing something?